R/get_meetnetten_visits.R

```diff
@@ -0,0 +1,169 @@
+#' @title Query visit data from Meetnetten
+#'
+#' @description This function queries the meetnetten database for visit
+#' data (data about a counting event) for a specified monitoring scheme or
+#' for all monitoring schemes within a specified species group. When no
+#' monitoring scheme or species group is specified, the visits of all monitoring
+#' schemes are returned.
+#'
+#' @param scheme_name the name of the monitoring scheme for which you want to
+#' extract visit data.
+#' @param species_group the name of the species group for which you want to
+#' extract visit data.
+#' @param connection dbconnection with the database 'S0008_00_Meetnetten'
+#' on the inbo-sql08-prd.inbo.be server
+#' @param collect If \code{FALSE} (the default), a remote tbl object is returned.
+#' This is like a reference to the result of the query but the full result of the
+#' query is not brought into memory. If \code{TRUE} the full result of the query
+#' is collected (fetched) from the database and brought into memory of the
+#' working environment.
+#'
+#' @return A remote \code{tbl} object (\code{collect} = \code{FALSE}) or a
+#' \code{tibble} dataframe (\code{collect} = \code{TRUE}) with following
+#' variables:
+#' \itemize{
+#'    \item \code{species_group}
+#'    \item \code{scheme}: the name of the monitoring scheme
+#'    \item \code{protocol}: the protocol used
+#'    \item \code{location}: the name of the location
+#'    \item \code{visit_id}: unique id for a count event
+#'    \item \code{validation_status}: validation status of the visit (visits
+#'    that are not approved are not provided)
+#'    \itemize{
+#'        \item \code{10}: visit not validated
+#'        \item \code{100}: visit approved
+#'        }
```

Comment:
Euh, wat is 'validated' en wat is 'approved'?  Ik zou denken dat een 'visit' dat 'not validated' is, zeker niet approved is?

Na even nadenken denk ik dat ik hem door heb, misschien er dit van maken?
```suggestion
#'    \item \code{validation_status}: validation status of the visit (visits
#'    that are validated but not approved are not provided)
#'    \itemize{
#'        \item \code{10}: visit not validated
#'        \item \code{100}: visit approved
#'        }
```

Reply:
valideren is het proces, goed/afkeuren is het resultaat van dat proces